### ISSUE TITLE
[node-red-contrib-state/shared-state] Error: Cannot find module 'is-valid-var-name'

### ISSUE THREAD

**Issue by mdegat01** [lorenwest/node-red-contrib-state#2]
Just installed version 1.4 and now I see this error in my logs:
```
21 Mar 10:29:09 - [warn] [node-red-contrib-state/shared-state] Error: Cannot find module 'is-valid-var-name'
Require stack:
- /config/node-red/node_modules/node-red-contrib-state/lib/state.js
- /opt/node_modules/@node-red/registry/lib/loader.js
- /opt/node_modules/@node-red/registry/lib/index.js
- /opt/node_modules/@node-red/runtime/lib/nodes/index.js
- /opt/node_modules/@node-red/runtime/lib/index.js
- /opt/node_modules/node-red/lib/red.js
- /opt/node_modules/node-red/red.js
```
Afterwards it then will not deploy my flows due to this error:
```
21 Mar 10:29:09 - [info] Waiting for missing types to be registered:
21 Mar 10:29:09 - [info]  - shared-state
```
Everything had been working fine prior to update, 1.4 seems to have some issue that prevents it from loading correctly.

**Comment by portalzine** [comment]
`npm install is-valid-var-name `

.. should solve that for you!

**Comment by mdegat01** [comment]
Yep, that did the trick.  Thanks!

**Comment by lorenwest** [comment]
Thank you for pointing out this bug. `is-valid-var-name` was improperly listed as a devDependency, and needed to be a dependency.  Fixed and published today in `1.4.1`.
